wikipedia_for_humans/__init__.py

```diff
@@ -1,16 +1,21 @@
 import wikipediaapi
-import requests
 from wikipedia_for_humans.util import match_one, fuzzy_match, \
     split_sentences, summarize, remove_parentheses, singularize
 from wikipedia_for_humans.exceptions import DisambiguationError
 from quebra_frases import word_tokenize, sentence_tokenize
+from requests_cache import CachedSession
+
+HEADERS = {'User-Agent', 'wikipedia_for_humans (https://github.com/OpenJarbas/wikipedia_for_humans)'}
+_SESSION = CachedSession(backend="memory", expire_after=60, headers=HEADERS)
+
 
 
 # internal handling of wikipediaapi
 def _get_page(page_name, lang="en", strict=False, auto_disambiguate=False):
     if isinstance(page_name, wikipediaapi.WikipediaPage):
         return page_name
     wiki_wiki = wikipediaapi.Wikipedia(lang)
+    wiki_wiki._session = CachedSession(backend="memory", expire_after=60, headers=HEADERS)
```

Comment:
Any reason this isn't just `_SESSION`?

Reply:
the parent package can close the session